reference-architecture/vmware-ansible/roles/static-ipaddress/tasks/main.yaml

```diff
@@ -0,0 +1,37 @@
+---
```

Comment:
What is the primary goal of this role? What does the additional static IP add that the normal IP of the host doesn't provide?